### ISSUE TITLE
Libation not discovering my Audible Library

### ISSUE THREAD

**Issue by glitteringdog** [rmcrackan/Libation#972]
**Describe the bug**
The program has failed to find my Library and nothing shows up in scanning results even when I login.
 
**To Reproduce**

1. Go to 'Settings'
2. Click on 'Accounts'
3. Scroll down to 'add account'
4. See error: 'Nothing shows, no books are scanned into Library'

**Expected behavior**
Books in my Audible account should be seen immediately after I add my Audible accounts and Login.

**Screenshots**

**Platform**

[Mac]

**Log Files**
Attach your Libation log file here.

**Comment by rmcrackan** [comment]
Sorry about this. Please attach the log file. Maybe that can shed some light.

**Comment by glitteringdog** [comment]
Thank you. I do not know how to locate the log files. I deleted all accounts and added just one, it worked.

**Comment by rmcrackan** [comment]
Thanks for following up. I'm glad it's fixed for you now.

**Comment by glitteringdog** [comment]
Hello, I am experiencing this issue again. 

This time, the audiobooks are not being seen, just as the issue above. 

I have tried scanning the library, but, nothing shows up. 

The passwords to the account, are correct. 

[_Log202410.zip](https://github.com/user-attachments/files/17467184/_Log202410.zip)
[_Log202409.zip](https://github.com/user-attachments/files/17467185/_Log202409.zip)
[_Log202407.zip](https://github.com/user-attachments/files/17467186/_Log202407.zip)

**Comment by glitteringdog** [comment]
There is also no way to import an .aax file on Macbook. 

Describe the bug
The program cannot import .aax files from my local files.

To Reproduce
Steps to reproduce the behavior:

Go to 'Import'
Click on 'Locate Audiobooks'
Scroll down to 'downloads'
See error: '.aax files cannot be clicked on'
Expected behavior
.aax files should be clickable to import into Libation.



Screenshots
If applicable, add screenshots to help explain your problem.

Platform

[Mac]

Log Files
Attach your Libation log file here. 
[_Log202410.zip](https://github.com/user-attachments/files/17467248/_Log202410.zip)
[_Log202409.zip](https://github.com/user-attachments/files/17467249/_Log202409.zip)
<img width="1085" alt="Screenshot 2024-10-22 at 00 47 10" src="https://github.com/user-attachments/assets/605ca4d1-db5b-4203-a06e-302f4e70f881">
<img width="1085" alt="Screenshot 2024-10-22 at 00 45 45" src="https://github.com/user-attachments/assets/b026e582-040e-4f90-b0b2-5d146930986f">

**Comment by glitteringdog** [comment]
> Thanks for following up. I'm glad it's fixed for you now.

Hi @rmcrackan the issue is happening again, I'd appreciate your kind assistance on this matter.

**Comment by glitteringdog** [comment]
[_Log202410.zip](https://github.com/user-attachments/files/17467336/_Log202410.zip)

**Comment by rmcrackan** [comment]
Thank you for your log files.

There appear to be several issues to address here so this is going to get long.

Firstly, I see a lot of scan errors in the log which look like the login didn't work. Were you able to get the login to work for each account? This is the main issue I'd like to help with.

Is your issue with the "locate audiobooks" feature? This feature is not what most people think it is and is confusing enough that I'm probably going to remove it. Locate audiobooks only works in the very specific case that you have the book in your audible library, and you have it in the folder you search, and the book's audiofile or folder contain the audible id (the ids look like this: B06WLMWF2S).

Regarding "There is also no way to import an .aax file". This is not something Libation can do.

Basically, although Libation has an iTunes-like look and feel, the intent is very different. The intended use is: scan your audible library, and download your books with the restrictive DRM/copy protection. Importing audiobooks from your computer or decrypting existing aax files is not part of it. That would be a very cool direction for it to go one day, but that's not currently what it's built for.

Also, to decrypt existing aax files, I recommend [https://github.com/rmcrackan/inAudible](inAudible). I haven't tried the mac version but the windows version worked well for me in the past.

**Comment by rmcrackan** [comment]
This (aax/audiobook import) is one of a number of far-reaching features that have been given a lot of thought for a long time with no commitments. I think it's fair to say we're not intending to do it in the near term.
